samples/apkman/sqlite.edl

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Open Enclave SDK contributors.
+// Licensed under the MIT License.
+
+enclave {
+    from "openenclave/edl/syscall.edl" import *;
+    from "platform.edl" import *;
+
+    trusted {
+        public int enc_main(int argc, [user_check] const char** argv);
```

Comment:
why do we need user_check here instead of doing a deep copy?

Reply:
I don't think we support deep copy of array of strings yet. I have removed argc and argv since it is not needed in this sample.